inspire/modules/workflows/tasks/submission.py

```diff
@@ -256,3 +256,15 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def add_note_entry(obj, eng):
+    """Add metadata to sip on approval."""
+    if was_approved(obj, eng):
```

Comment:
I added it before adding the task to the `workflow_if(was_approved)` and then I forgot to amend it. ;)